web/pages/feeds/categories.svelte

```diff
@@ -85,6 +86,7 @@
                         params: { feed: feed.id },
                         route: 'categories'
                       })}
+                      on:click={showSecondStep}
```

Comment:
Calling `showSecondStep` on click is not a good idea, because URL is the source of truth, not clicks.

Here is a use case when it will not work:

1. I open categories page
2. I click on feed A
3. Slowreader open feed A and I see Back button in the menu
4. I press Back button *in the browser*
5. I still see Back button in navbar

Reply:
I will try to figure out how to implement back routes.